data/templates/logs/logrotate/vyos-atop.tmpl

```diff
@@ -0,0 +1,20 @@
+/var/log/atop/atop.log {
+    daily
+    dateext
+    dateformat _%Y-%m-%d_%H-%M-%S
+    maxsize {{ max_size|default('10') }}M
```

Comment:
As there is a `<defaultValue>` specified on the XML implementation the `default()` statement is superfluous.

Reply:
Fixed.